lib/sendgrid_helper.rb

```diff
@@ -24,4 +24,23 @@ def self.handle_response(url)
   rescue Curl::Err::CurlError, JSON::ParserError => e
     false
   end
+
+  def self.smtp_alive?(host, port)
+    Net::SMTP.start(host, port) do |smtp|
+      smtp.enable_starttls_auto
+      smtp.ehlo(Socket.gethostname)
+      smtp.finish
+    end
+    true
+  rescue StandardError => e
+    false
+  end
+
+  def self.messagelabs_alive?(host='cluster2.gsi.messagelabs.com')
+    smtp_alive?(host, 25)
+  end
+
+  def self.sendgrid_alive?(host='smtp.sengrid.net')
+    smtp_alive?(host, 587)
+  end
```

Comment:
It looks like these two functions `messagelabs_alive?` and `sendgrid_alive?` aren't used so we should remove them